inc/define.php

```diff
@@ -463,7 +463,7 @@
     'priority_6', 'refresh_views', 'set_default_tech',
     'set_default_requester', 'show_count_on_tabs',
     'show_jobs_at_login', 'task_private', 'task_state',
-    'use_flat_dropdowntree', 'palette', 'page_layout',
+    'use_flat_dropdowntree', 'use_flat_dropdowntree_on_display', 'palette', 'page_layout',
```

Comment:
The name of the criteria doesn't reflect the fact that it only impact search results.

Reply:
Renamed to ```use_flat_dropdowntree_on_search_result```